dockerfiles/KerasContainerDockerfile

```diff
@@ -0,0 +1,14 @@
+ARG REGISTRY
+ARG CODE_VERSION
+ARG RPC_VERSION
+FROM ${REGISTRY}/${RPC_VERSION}-rpc:${CODE_VERSION}
+
+RUN pip install -q keras
+RUN pip install -q tensorflow
+RUN pip install -q numpy==1.16 # MXNet requires numpy < 1.15
```

Comment:
MXNet requires `numpy<=1.15.2,>=1.8.2`.
https://github.com/apache/incubator-mxnet/blob/master/python/setup.py#L33

Reply:
I will remove the comment. I guess mxnet 1.4.0 requires <1.15 (because I saw from my pip). This part will keep changing as mxnet 1.4.* is upgraded.